<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduinoConnectionDlg</class>
 <widget class="QDialog" name="ArduinoConnectionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configure your Arduino</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="port" stretch="1,2">
     <item>
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="portCbx">
       <property name="maxVisibleItems">
        <number>8</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="baudrate" stretch="1,2">
     <item>
      <widget class="QLabel" name="baudrateLabel">
       <property name="text">
        <string>Baudrate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="baudrateCbx">
       <property name="maxVisibleItems">
        <number>8</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
